supervisor/backups/manager.py

```diff
@@ -233,6 +236,9 @@ async def _do_restore(
 
                 if FOLDER_HOMEASSISTANT in folder_list:
                     backup.restore_homeassistant()
```

Comment:
Not really related, but this should on `if homeassistant:` (L250) instead of here

Reply:
Good point. I remember asking myself that when I did the backup/restore de-duplication PR, but somehow forgot to follow up on it.